lib/bundler/plugin/v1/command.rb

```diff
@@ -0,0 +1,22 @@
+module Bundler
+  module Plugin
+    module V1
+      class Command
+        attr_reader :command_name, :command_short_description, :command_long_description
+
+        def initialize
+          @command_name = "Plugin [OPTIONS]"
+          @command_short_description = "Unimplemented short description"
+          @command_long_description = "Unimplemented long description"
+        end
+
+        def run(options, args)
+          Bundler.ui.info "The plugin hasn't implemented the run method"
```

Comment:
I was thinking that improperly implemented plugin shouldn't be allowed to crash bundler. That's why some errors were "swallowed". 

Of course, we can let bundler crash and let the user decide to uninstall the misbehaving plugin. Now that I think about it, it does make sense.